app.py

```diff
@@ -38,6 +38,22 @@ def post_random_game_from_users_collection() -> Union[str, Response]:
         return abort(Response(response=str(exc), status=500))
 
 
+@app.route("/all_games")
+def post_all_games_from_users_collection() -> Union[str, Response]:
```

Comment:
nit: why do you call this view `post_all_games_from_users_collection`? when i read this i think the request method would be POST, but it's GET

Reply:
Appreciate the nits!  So my thought process was that this function is _posting_ all the games from a users collection to the end point "all_games".  What makes the request method get?  I guess because it's calling a function from bgg_companion_api?

Reply:
it could be me getting tripped up on how you're naming things, but this view is processing GET requests for `/all_games` routes. i thought it made more sense to call this `get_all_games_from_users_collection` because the view is getting data to display back to the user.

i have no preference for how you name things, if it makes sense to you then feel free to leave this as-is

Reply:
No I certainly would rather use a naming convention that makes for more readable code to others.  And the fact that it's confusing to you is definitely a red flag.  

So you think it makes more sense that the function is make GET requests via the function calls for the all_games endpoint, rather than the function is POSTing information it called to the all_games endpoint?

Reply:
Discussed and get makes more sense!